src/components/Card/docs/index.story.tsx

```diff
@@ -77,6 +81,7 @@ export default {
           ...[
             { title: 'Side By Side', source: examples.sideBySide },
             { title: 'Stacked', source: examples.stacked },
+            { title: 'Stacked On Mobile', source: examples.stackedMobile },
             { title: 'Ratios', source: examples.ratio },
             { title: 'Flipped Ratios', source: examples.flippedRatio },
             {
```

Comment:
Do we need an example for `invertInfoPosition` and `mobile` maybe?

Reply:
by the design system we should not allowed invertInfoPosition on mobile